core/config.go

```diff
@@ -11,6 +11,8 @@ import (
 )
 
 const configEnvVar = "CHANGIE_CONFIG_PATH"
+const DefaultFileMode os.FileMode = 0644
+const DefaultDirMode os.FileMode = 0755
```

Comment:
```suggestion
const CreateFileMode os.FileMode = 0644
const CreateDirMode os.FileMode = 0755
```
I think naming these after what we are doing makes more sense then default, as to me default implies there are other options which there are not.